init/src/main/groovy/com/taulia/boundaries/Transmission.groovy

```diff
@@ -4,7 +4,15 @@ package com.taulia.boundaries
 class Transmission {
   Gear gear
   Moveable moveable
-  Transmission(Moveable moveable){
+
+  Transmission(Moveable moveable) {
     this.moveable = moveable
   }
+
+  def shift(Gear gear) {
+    if (this.gear == Gear.DRIVE && this.moveable.currentSpeedInMph() != 0 && gear == Gear.PARK) {
```

Comment:
would this ever work?  Should it be an or?  Is there a test for both conditions?